### DISCUSSION TITLE
New Tab for each Page Menu(similar to IFrame Plugin in AdminLTE v3)

### DISCUSSION THREAD

**Discussion by mulia-dhamma** [primefaces/community#1255] [PrimeFaces Templates]
Is it possible for Pandora Layout to open a page to a new tab in a header ? so user can change to the another page in existing default layout by using tab without using (open link in new tab).
Example: IFrame Plugin in Admin LTE V3

**Comment by tandraschko** [comment]
for now we decided to only maintain following templates:

          - "apollo"
          - "atlantis"
          - "avalon"
          - "diamond"
          - "freya"
          - "poseidon"
          - "ultima"
          - "verona"
